client/src/components/PageExplorer/PageExplorerPanel.tsx

```diff
@@ -143,7 +143,7 @@ class PageExplorerPanel extends React.Component<PageExplorerPanelProps, PageExpl
 
     return (
       <Transition name={transition} className="c-page-explorer" component="nav" label={STRINGS.PAGE_EXPLORER}>
-        <div key={depth} className="c-transition-group">
+        <div key={depth} className="c-transition-group" dir="ltr">
```

Comment:
Question : Is it out of the scope of the current PR ? Is there a dedicated issue for to support RTL in the explorer ? 
I tried setting `direction: rtl` on the explorer and it does not appear to be a massive change, I would be happy to work on this if no work has started yet on this issue 

![image](https://user-images.githubusercontent.com/1702255/149473369-8607570d-7dd7-4a99-a382-444297c4c1d1.png)

Reply:
Yep, I left this out for now (or I reckon this PR will be massive!), but there isn't a dedicated issue for this yet

Reply:
Looks like this PR needs rebasing on `main` after #7844, plus fixing the merge conflicts.

Reply:
I would expect this to require a lot of re-work, as our preferred approach for RTL support will be to switch to CSS logical properties (https://github.com/wagtail/wagtail/discussions/7793) now that we don’t have support for IE11.

The good news is that switch to logical properties should largely be a matter of search-and-replace.